Disher/View Controllers/LoginViewController.m

```diff
@@ -63,6 +63,8 @@ - (void) registerUser {
     PFUser *newUser = [PFUser user];
     newUser.username = self.usernameField.text;
     newUser.password = self.passwordField.text;
+    NSArray *array = @[];
```

Comment:
ditto: NSArray<YourType>

moreover, looks like you can just inline this below :)